gpu-z@2.37.0: download via aria2 failed

Updating 'gpu-z' (2.36.0 -> 2.37.0)
Downloading new version
Starting download with aria2 ...
Download: 02/22 20:49:54 [ERROR] CUID#8 - Download aborted. URI=http://nl1-dl.techpowerup.com/files/GPU-Z.2.37.0.exe#/GPDownload: Download Results:                                                                                   .exe
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: b4372a|ERR |       0B/s|C:/Scoop/cache/gpu-z#2.37.0#http_nl1-dl.techpowerup.com_files_GPU-Z.2.37.0.exe_GPU-Z.exe
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR http://nl1-dl.techpowerup.com/files/GPU-Z.2.37.0.exe#/GPU-Z.exe
    referer=http://nl1-dl.techpowerup.com/files/GPU-Z.2.37.0.exe#/
    dir=C:\Scoop\cache
    out=gpu-z#2.37.0#http_nl1-dl.techpowerup.com_files_GPU-Z.2.37.0.exe_GPU-Z.exe

ERROR & 'C:\Scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Scoop\cache\gpu-z.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/5.1 (Windows NT 10.0; Win64; x64; Desktop)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=10 --max-connection-per-server=10 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=13984 --continue --summary-interval 0

--- a/bucket/gpu-z.json
+++ b/bucket/gpu-z.json
@@ -3,7 +3,7 @@
     "description": "Lightweight system utility designed to provide vital information about your video card and graphics processor.",
     "homepage": "https://www.techpowerup.com/gpuz/",
     "license": "Freeware",
-    "url": "http://nl1-dl.techpowerup.com/files/GPU-Z.2.37.0.exe#/GPU-Z.exe",
+    "url": "http://us2-dl.techpowerup.com/files/GPU-Z.2.37.0.exe#/GPU-Z.exe",
     "hash": "md5:d9f45181b6df70ef02205eff6d1a9132",
     "bin": "GPU-Z.exe",
     "shortcuts": [
@@ -17,7 +17,7 @@
         "regex": "TechPowerUp GPU-Z v([\\d.]+)"
     },
     "autoupdate": {
-        "url": "http://nl1-dl.techpowerup.com/files/GPU-Z.$version.exe#/GPU-Z.exe",
+        "url": "http://us2-dl.techpowerup.com/files/GPU-Z.$version.exe#/GPU-Z.exe",
         "hash": {
             "url": "https://www.techpowerup.com/download/techpowerup-gpu-z/",
             "regex": "(?sm)$basename.*?$md5"

